code/modules/admin/panicbunker.dm

```diff
@@ -0,0 +1,37 @@
+var/list/panic_bunker_bypass = list()
+
+/datum/admins/proc/panicbunker()
+	set category = "Server"
+	set name = "Toggle Panic Bunker"
+
+	if(!check_rights(R_SERVER))
+		return
+
+	config.panic_bunker = (!config.panic_bunker)
+
+	log_and_message_admins("[key_name(usr)] has toggled the Panic Bunker, it is now [(config.panic_bunker ? "on" : "off")]")
+	if (config.panic_bunker && (!dbcon || !dbcon.IsConnected()))
+		message_admins("The Database is not connected! Panic bunker will not work until the connection is reestablished.")
+
+/datum/admins/proc/addbunkerbypass(ckeytobypass as text)
+	set category = "Server"
+	set name = "Add Panic Bunker Bypass"
+	set desc = "Allows a given ckey to connect despite the panic bunker for a given round."
+
+	if(!check_rights(R_SERVER))
+		return
+
+	global.panic_bunker_bypass |= ckey(ckeytobypass)
+
+	log_admin("[key_name(usr)] has added [ckeytobypass] to the current round's bunker bypass list.")
+	message_admins("[key_name_admin(usr)] has added [ckeytobypass] to the current round's bunker bypass list.")
+
+/datum/admins/proc/revokebunkerbypass(ckeytobypass as text)
```

Comment:
`ckeytobypass in global.panic_bunker_bypass` would let them just pick one to remove I believe.